src/constants/imgUrlValidate.ts

```diff
@@ -0,0 +1,22 @@
+import { z } from 'zod';
+
+export const urlRegex = /^https:\/\/(images\.unsplash\.com|i\.imgur\.com|unsplash\.com)\/.+/;
+
+export function imgUrlValidate(url: string): string[] | null {
+  const schema = z.string().url({ message: '請填入正確的網址格式, ex: https://www.journeybites.com' }).refine(url => url.startsWith('https://'), {
+    message: 'URL must start with https://'
+  });

```

Comment:
```suggestion
  const schema = z.string().url({ message: '請填入正確的網址格式, ex: https://www.journeybites.com' }).or(z.literal(''));

```
zod url 就會檢查 https 了